setup.py

```diff
@@ -45,26 +44,30 @@ def run_tests(self):
 
 class BuildDocsCommand(setuptools.command.build_py.build_py):
     apidoc_command = (
-        'sphinx-apidoc', '-f', '-o', 'docs', '--no-toc', 'coalib'
+        'sphinx-apidoc', '-f', '-o', 'docs',
```

Comment:
Shouldn't it be placed with line 46?
> See https://github.com/coala/coala-bears/blob/5ca3eb7/setup.py#L28

Reply:
I'm guessing earlier it was because of the line length, now it can be on the same line.

Reply:
See earler https://github.com/coala/coala/pull/4897#discussion_r159525549